### DISCUSSION TITLE
Chrome extension

### DISCUSSION THREAD

**Discussion by itzanway** [community/community#69074] [Programming Help]
### Body

Can anyone help me in displaying the youtube likes and dislikes by hover on focus using a chrome extension

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by vishalkrishna8** [comment]
This involves steps like setting up the extension, injecting JavaScript, fetching data from the YouTube API, displaying the information, handling events, testing, and publishing. It's a complex task that requires coding skills and knowledge of extension development.

**Reply by python-nerd-3** [reply]
No offense but you make it sound intimidating and overengineered, you can just get the number from the API and slap it onto a HTML label tag.

**Comment by cmdhell** [comment]
It's a complex task that requires coding skills and knowledge of extension development. If you have a good knowledge then you can do it.
